src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.cs

```diff
@@ -628,7 +628,11 @@ private static CultureData CreateCultureWithInvariantData()
             // all available calendar type(s).  The first one is the default calendar
             invariant._waCalendars = new CalendarId[] { CalendarId.GREGORIAN };
 
+#if TARGET_BROWSER
+            if (!GlobalizationMode.InvariantFast)
```

Comment:
This feels wrong to me but I'm not sure how to fix it. Maybe a common helper property that has the ifdef encapsulated in it, with a name that describes the logic behind this?

Reply:
+1